src/modules/roc_fec/reader.cpp

```diff
@@ -82,26 +79,43 @@ packet::PacketPtr Reader::read_() {
                 source_ = pp->rtp()->source;
                 has_source_ = true;
             }
+
             cur_sbn_ = pp->fec()->source_block_number;
+            resize_(pp->fec()->source_block_length);
```

Comment:
I suggest to extract the body of the `if (!started_) {` block into a method.

Reply:
Agree.

Reply:
https://github.com/roc-project/roc/pull/174#discussion_r283099439

Reply:
Done